src/zenml/models/stack_models.py

```diff
@@ -153,13 +152,10 @@ def is_valid(self) -> bool:
         Returns:
             True if the stack is valid, False otherwise.
         """
-        if (
-            StackComponentType.ARTIFACT_STORE
-            and StackComponentType.ORCHESTRATOR in self.components.keys()
-        ):
-            return True
-        else:
-            return False
+        return (
+            StackComponentType.ARTIFACT_STORE in self.components
+            and StackComponentType.ORCHESTRATOR in self.components
+        )
```

Comment:
I'd suggest tomove this into the `StackBaseModel` instead of duplicating it here

Reply:
If I remember correctly I tried doing that, but the `components` attribute doesn't exist on the `StackBaseModel` but only on those subclasses (with type `Dict[StackComponentType, List[UUID]]` and `Dict[StackComponentType, List[ComponentResponseModel]]`)

Reply:
ah I see, nevermind then